test/e2e/common/host_path.go

```diff
@@ -97,7 +97,7 @@ var _ = Describe("[sig-storage] HostPath", func() {
 		source := &v1.HostPathVolumeSource{
 			Path: "/tmp",
 		}
-		pod := testPodWithHostVol(volumePath, source, true)
+		pod := testPodWithHostVol(volumePath, source, false)
```

Comment:
After applying this change, all callers of testPodWithHostVol() pass false as privileged argument.
How about removing the argument for cleanup?